builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -606,7 +615,7 @@ function store.update_paths()
 	pkgmgr.refresh_globals()
 	for _, mod in pairs(pkgmgr.global_mods:get_list()) do
 		if mod.author and mod.release > 0 then
-			local id = mod.author:lower() .. "/" .. mod.name
+			local id = mod.author:lower():encode_uri_component() .. "/" .. mod.name
```

Comment:
id is used to track packages locally, and should never be used in a URL as case info is lost